docker/Parsl/Dockerfile

```diff
@@ -14,6 +14,6 @@ FROM ${BASE_IMAGE}
 # the rust build, so we'll use that until a more sensible solution
 # is found.
 
-run python3 -m pip install parsl
+RUN python3 -m pip install parsl
```

Comment:
Our validator says the commit message head exceed the set threshold:

![Screen Shot 2021-07-19 at 7 34 13 AM](https://user-images.githubusercontent.com/862123/126177650-456ff2ba-af1f-4e23-8973-d8c304611113.png)

Reply:
I tried to edit that commit message, and it is still there.. do you have any suggestions?

Reply:
with Stephen's help issue is resolved